uber/models.py

```diff
@@ -474,8 +474,8 @@ def jobs_for_signups(self):
 
         def guess_attendee_watchentry(self, attendee):
             return self.query(WatchList).filter(and_(or_(WatchList.first_names.contains(attendee.first_name),
-                                                         WatchList.email == attendee.email,
-                                                         WatchList.birthdate == attendee.birthdate),
+                                                         and_(WatchList.email != '',WatchList.email == attendee.email),
```

Comment:
Our ``pep8`` style checker is failing on this line because you don't have a space after the comma between the two arguments to ``and_``.